plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -11,6 +11,8 @@ banana:
 hydra:
   sweeper:
     params:
+      max_batch_size: null
+  
```

Comment:
I don't think you need to add it here, adding to the one inside the plugin should be enough.